packages/fcl-core/src/wallet-utils/on-message-from-fcl.js

```
@@ -16,12 +16,12 @@ export const onMessageFromFCL = (messageType, cb = () => {}) => {
   }
 
   const internal = e => {
-    const {data, origin} = e
+    const {data} = e
     if (typeof data !== "object") return
     if (typeof data == null) return
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)